src/main/java/edu/cornell/kfs/pmw/batch/service/PaymentWorksVendorDataProcessingIntoKfsService.java

```diff
@@ -11,7 +11,7 @@
 
 public interface PaymentWorksVendorDataProcessingIntoKfsService {
     
-    boolean createValidateAndRouteKFSVendor(PaymentWorksVendor savedStgNewVendorRequestDetailToProcess, Map<String, List<PaymentWorksIsoFipsCountryItem>> paymentWorksIsoToFipsCountryMap,
+    boolean createValidateAndRouteOrSaveKFSVendor(PaymentWorksVendor savedStgNewVendorRequestDetailToProcess, Map<String, List<PaymentWorksIsoFipsCountryItem>> paymentWorksIsoToFipsCountryMap,
```

Comment:
I noticed that these changes will cause the saved and enroute documents to be lumped into the same combined category in the reports. Should the reports have some kind of separation between the saved and enroute docs, to make it easier for the functionals to tell which is which? (Not sure if this method signature needs updating to assist with that.)

Reply:
I will check with functionals if there should be a separate report section for saved documents instead of updating the report section label.